lib/xsv/sax_parser.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Xsv
+  class SaxParser
+    ATTR_REGEX = /((\S+)="(.*?)")/m.freeze
+
+    def parse(io)
+      @start_element = respond_to?(:start_element) ? method(:start_element) : nil
```

Comment:
You can drop the ternaries here and juts use `respond_to?(...) && method(...)`, unless it's important that these are nil and not false. In which case I believe `...if respond_to?()` would work.

Reply:
Good point. I think we can drop the check altogether since all implementations probably implement all callbacks anyway.

Reply:
Fixed!